src/ip/Address.h

```diff
@@ -44,10 +44,10 @@ class Address
     /** @name Constructors */
     /*@{*/
     Address() { setEmpty(); }
+    Address(const struct sockaddr_storage &);
```

Comment:
By default, please disallow implicit conversions in new code:

```suggestion
    explicit Address(const struct sockaddr_storage &);
```